Trakt/Extensions.cs

```diff
@@ -121,7 +121,38 @@ public static bool MetadataIsDifferent(this TraktMovieCollected collectedMovie,
                || collectedMovie.Metadata.Resolution != resolution
                || collectedMovie.Metadata.Is3D != is3D
                || collectedMovie.Metadata.Hdr != hdr
-               || collectedMovie.Metadata.MediaType != null;
+               || collectedMovie.Metadata.MediaType != TraktMediaType.digital;
+    }
+
+    /// <summary>
+    /// Checks if metadata of new collected movie is different from the already collected.
+    /// </summary>
+    /// <param name="collectedEpisode">The <see cref="TraktEpisodeCollected"/>.</param>
+    /// <param name="episode">The <see cref="Movie"/>.</param>
+    /// <returns><see cref="bool"/> indicating if the new episode has different metadata to the already collected.</returns>
+    public static bool MetadataIsDifferent(this TraktEpisodeCollected collectedEpisode, Episode episode)
+    {
+        var audioStream = episode.GetMediaStreams().FirstOrDefault(x => x.Type == MediaStreamType.Audio);
```

Comment:
`GetMediaStreams` also contains the default video stream, but you are calling them thrice (combined). There's no caching afaict.